src/System.Data.Common/tests/System/Data/XmlDataReaderTest.cs

```
@@ -270,45 +268,33 @@ public CustomTypeXml(string str)
             {
                 XmlDocument doc = new XmlDocument();
                 doc.LoadXml(str);
-                _mFuncXmlNode = doc.DocumentElement;
+                Node = doc.DocumentElement;
             }
 
             public CustomTypeXml(XmlNode xNode)
             {
-                _mFuncXmlNode = xNode;
+                Node = xNode;
             }
             #endregion
 
-            #region Node (set/get)
-            public XmlNode Node
-            {
-                get
-                {
-                    return _mFuncXmlNode;
-                }
-                set
-                {
-                    _mFuncXmlNode = value;
-                }
-            }
-            #endregion
-            #region ToString
+            public XmlNode Node { get; set; }
+
             public override string ToString()
             {
                 return Node.OuterXml;
             }
-            #endregion
 
             /* IXmlSerializable overrides */
             #region WriteXml
             void IXmlSerializable.WriteXml(XmlWriter writer)
             {
                 XmlDocument doc = new XmlDocument();
-                doc.LoadXml(_mFuncXmlNode.OuterXml);
+                doc.LoadXml(Node.OuterXml);
 
                 // On function level
                 if (doc.DocumentElement.Name == "Func")
                 {
+                    //TODO: what
                     try { doc.DocumentElement.Attributes.Remove(doc.DocumentElement.Attributes["ReturnType"]); }
```

swallowing all exceptions?